infrastructure/aws/web-serve.sh

```diff
@@ -18,10 +18,15 @@ MOZSEARCH_PATH=$(dirname "$SCRIPT_PATH")/..
 CONFIG_REPO=$(readlink -f $1)
 CONFIG_INPUT="$2"
 
+# The EBS volume will no longer be mounted at /dev/xvdf but instead at an
+# arbitrarily assigned nvme id.  However, since we only have a single EBS volume
+# and we dynamically attach it, we're pretty certain what the ID will be:
+EBS_NVME_DEV=nvme1n1
```

Comment:
Not clear to me why this isn't `nvme2n1` - won't 1n1 be the local storage on the web server instance? Or does the web instance not get that?

Reply:
t3.large doesn't have instance storage.  The new naming scheme appears to be that machines with instance storage have a "d" suffix.  So "m5.large" has no disk, "m5d.large" does have a disk, etc.  Only our "c5d.2xlarge" indexer and "c5d.2xlarge" AMI provisioners have disks.

Reply:
Ok, makes sense then.